src/jsx/modal.jsx

```diff
@@ -11,25 +11,36 @@ class ModalMessage extends Component {
         this.state = {
```

Comment:
You have to look into the OpenSource guidelines once more, but I think you have to add the license header also to each source file.